CHANGELOG.md

```diff
@@ -1,3 +1,38 @@
+## 0.23.0 (September 22, 2025)
+
+NOTES:
+
+* **This release requires Juju controller version 2.9.49 or higher Juju.**
+* **If using JAAS, this release requires Juju controller version 3.6.5 or higher.**
+* This release uses Juju client api code from the Juju 3.6.4 release.
+
+ENHANCEMENTS
+
+* An optional warning instead of an error on failed resource deletion by @kian99 in [#877](https://github.com/juju/terraform-provider-juju/pull/877)
+* Allow changing charm channel and revision together by @luci1900 in [#899](https://github.com/juju/terraform-provider-juju/pull/889)
+* Avoiding in-place updates of relations by @kian99 in [#898](https://github.com/juju/terraform-provider-juju/pull/898)
+
+BUG FIXES
+
+* Fix for [#881](https://github.com/juju/terraform-provider-juju/issues/881) by @SimoneDutto in [#890](https://github.com/juju/terraform-provider-juju/pull/890)
+* Fix for the offer and integration resource logic by @kian99 in [#893](https://github.com/juju/terraform-provider-juju/pull/893)
+* Fix for reading offer resources by @SimoneDutto in [#906](https://github.com/juju/terraform-provider-juju/pull/906)
+
```

Comment:
https://github.com/juju/terraform-provider-juju/issues/473 and https://github.com/juju/terraform-provider-juju/issues/235 should be fixed thanks to https://github.com/juju/terraform-provider-juju/pull/898 which was already mentioned under enhancements.

Reply:
moved #898 to fixes